src/column-layout/styles.scss

```diff
@@ -130,5 +131,49 @@ div.column-layout {
     &.grid-horizontal-borders {
       @include make-horizontal-borders;
     }
+
+    // Alternative implementation based on minimum column widths and CSS Grid
+    &.css-grid {
+      display: grid;
+      grid-template-columns: repeat(var(#{custom-props.$columnLayoutColumnCount}, 1), 1fr);
+
+      pointer-events: none;
+
+      /* stylelint-disable-next-line selector-max-universal */
+      &.grid-variant-text-grid > * {
```

Comment:
Can we assign a class name (`.column`) to the items, instead of fighting with stylelint?

Reply:
Sure!